nix/deps/gradle/generate.sh

```diff
@@ -1,18 +1,16 @@
 #!/usr/bin/env bash
 
-# Stop on any failures, including in pipes
-set -e
-set -o pipefail
+# This script takes care of generating/updating the maven-sources.nix file
```

Comment:
How should I call this script correctly? Right now it doesn't work for me, I'm most likely missing something.

```bash
cd status-mobile/
make shell
nix/deps/gradle/generate.sh

# => Regenerating Nix files...
# => ./gradlew: line 25: exec: gradle: not found
```

Reply:
Default shell does not have Gradle:
https://github.com/status-im/status-mobile/blob/31ca94dbfa441e2bfbdceb53ecac2526aaa31a54/nix/shell.nix#L10-L27
There is a dedicated shell for using just Gradle:
https://github.com/status-im/status-mobile/blob/31ca94dbfa441e2bfbdceb53ecac2526aaa31a54/nix/shells.nix#L42-L52
And there's a dedicated target that uses that shell and the `generate.sh` script:
https://github.com/status-im/status-mobile/blob/31ca94dbfa441e2bfbdceb53ecac2526aaa31a54/Makefile#L96-L98
So the stock way of using this is just doing `make nix-update-gradle`.

Reply:
Thank you, I'm gonna play around another time.